### PR TITLE
Denormalize tax class for order lines and orders

### PR DESCRIPTION
**Pull request by maarcingebala**
TaxClass instance needs to be denormalized for order lines to keep the tax rate information even when tax rate is removed from the system.

# Impact

* [ ] New migrations
* [ ] New/Updated API fields or mutations
* [ ] Deprecated API fields or mutations
* [ ] Removed API types, fields, or mutations
* [ ] Documentation needs to be updated

# Pull Request Checklist

* [ ] Privileged queries and mutations are guarded by proper permission checks
* [ ] Database queries are optimized and the number of queries is constant
* [ ] Database migration files are up to date
* [ ] The changes are tested
* [ ] GraphQL schema and type definitions are up to date
* [ ] Changes are mentioned in the changelog

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-6391 Simple taxes](https://app.clickup.com/t/2549495/SALEOR-6391)

**Review by IKarbowiak — APPROVED** [review]

File: saleor/tax/utils.py [hunk 1]
```diff
@@ -179,6 +179,11 @@ def normalize_tax_rate_for_db(tax_rate: Decimal) -> Decimal:
     return tax_rate / 100
 
 
+def denormalize_tax_rate_from_db(tax_rate: Decimal) -> Decimal:
+    # Revert results of `normalize_tax_rate_for_db`.
+    return tax_rate * 100
```
Comment (korycins) [inline saleor/tax/utils.py:184]: Question: shouldn't we apply a `decimal.quantize` here?
```
In [2]: Decimal(0.22)*100
Out[2]: Decimal('22.00000000000000011102230246')
```

File: saleor/tax/utils.py [hunk 2]
```diff
@@ -179,6 +179,11 @@ def normalize_tax_rate_for_db(tax_rate: Decimal) -> Decimal:
     return tax_rate / 100
```
Comment (korycins) [inline saleor/tax/utils.py:179]: I know, that this is not a scope of this PR, but shouldn't we have a protection here for divide by zero?
something like:
```
if tax_rate == 0:
    return 0
```

**Review by korycins — APPROVED** [review]

**Comment by korycins** [comment]
Also @maarcingebala  question. Shouldn't we add labels to the new fields? ADDED_IN_X, PREVIEW_FEATURE?

**Review by SzymJ — APPROVED** [review]

File: saleor/tax/utils.py (hunk 1 above)
Comment (maarcingebala) [reply saleor/tax/utils.py:184]: Hmm I think this is a good idea.

Comment (maarcingebala) [reply saleor/tax/utils.py:184]: I'm checking this and it's not `Decimal(0.22)` but `Decimal("0.22")`. Any multiplication on that seems to give correct values:
```
ipdb> order.shipping_tax_rate
Decimal('0.2300')
ipdb> order.shipping_tax_rate * 100
Decimal('23.0000')
```

Comment (patrys) [reply saleor/tax/utils.py:184]: We don't convert floats to decimals, there is no need to do any quantization or rounding here.

File: saleor/tax/utils.py (hunk 2 above)
Comment (maarcingebala) [reply saleor/tax/utils.py:179]: We're dividing by 100 always here, `0/100` is allowed ;)

**Review by eCzerniak — APPROVED** [review]

**Pull request merged**